.github/workflows/helm-release.yaml

```diff
@@ -0,0 +1,41 @@
+name: helm-release
+
+on:
+  push:
+    tags: ['*']
```

Comment:
Can you change this this to match this?:

https://github.com/ansible/awx-operator/blob/6b873b05abe37f39d90a8594abc54a899d210b06/.github/workflows/promote.yaml#L4-L5